templates/wordpress/1/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '2'
+services:
+  mariadb:
+      image: 'bitnami/mariadb:latest'
+      volumes:
+        - 'mariadb_data:/bitnami'
+      environment:
+        - MARIADB_USER=bn_wordpress
+        - MARIADB_DATABASE=bitnami_wordpress
+        - ALLOW_EMPTY_PASSWORD=yes
```

Comment:
All the info is hardcoded, please use questions in the rancher-compose, with these as default values.